megapowerani.com.template.json

```diff
@@ -0,0 +1,70 @@
+{
+    "providerId": "megapowerani.com",
+    "providerName": "Freshsales Test",
+    "serviceId": "template",
+    "serviceName": "Domain verification",
```

Comment:
Can you sync `serviceId` with `serviceName`? "template" seems to be very generic and "verification" already quite specific.

Reply:
I will have this updated._